helper/helper.pro

```diff
@@ -2,7 +2,7 @@ TEMPLATE = app
 
 QT += core
 
-LIBS += -lisomd5
+LIBS += -lcheckisomd5 -limplantisomd5
```

Comment:
I wouldn't call this a "proper" dependency resolution but it's a good start. The right way would (for example but probably the best way right nwo) probably be shipping a pkg-config file with the implantisomd5 library and then use pkg-config instead of this. It will then make compiling against libraries in a nonstandard prefix work and it also makes distro differences smoother.